modules/sara_r4/cellular_r4_api.c

```diff
@@ -589,7 +559,7 @@ CellularError_t Cellular_SocketRecv( CellularHandle_t cellularHandle,
     }
     else
     {
-        sessionId = ( uint32_t ) socketHandle->pModemData;
+        sessionId = _Cellular_GetSessionId( pContext, socketHandle->socketId );
```

Comment:
The previous implementation never works or due to the changes in FreeRTOS-Cellular-Interface?

Reply:
The previous implementation is not comply with C90. The function _Cellular_GetSessionId is defined in module implementation. It should be changed in one PR for all the function name in this module.